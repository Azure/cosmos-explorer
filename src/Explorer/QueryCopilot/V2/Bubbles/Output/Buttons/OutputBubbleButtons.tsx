import { Stack } from "@fluentui/react";
import { CopyButton } from "Explorer/QueryCopilot/V2/Bubbles/Output/Buttons/Copy/CopyButton";
import { FeedbackButtons } from "Explorer/QueryCopilot/V2/Bubbles/Output/Buttons/Feedback/FeedbackButtons";
import { InsertButton } from "Explorer/QueryCopilot/V2/Bubbles/Output/Buttons/Insert/InsertButton";
import { MoreButton } from "Explorer/QueryCopilot/V2/Bubbles/Output/Buttons/More/MoreButton";
import React from "react";

export const OutputBubbleButtons = ({ sqlQuery }: { sqlQuery: string }): JSX.Element => {
  return (
    <Stack horizontal>
      <Stack.Item style={{ paddingTop: "5px" }}>
        <InsertButton sqlQuery={sqlQuery} />
      </Stack.Item>
      <Stack.Item>
        <CopyButton sqlQuery={sqlQuery} />
      </Stack.Item>
      <Stack.Item>
        <FeedbackButtons sqlQuery={sqlQuery} />
      </Stack.Item>
      <Stack.Item>
        <MoreButton />
      </Stack.Item>
    </Stack>
  );
};
