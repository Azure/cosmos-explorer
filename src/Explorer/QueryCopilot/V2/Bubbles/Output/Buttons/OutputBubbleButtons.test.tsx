import { OutputBubbleButtons } from "Explorer/QueryCopilot/V2/Bubbles/Output/Buttons/OutputBubbleButtons";
import { shallow } from "enzyme";
import React from "react";

describe("Output Bubble Buttons snapshot tests", () => {
  it("should render", () => {
    const wrapper = shallow(<OutputBubbleButtons sqlQuery={""} />);

    expect(wrapper).toMatchSnapshot();
  });
});
