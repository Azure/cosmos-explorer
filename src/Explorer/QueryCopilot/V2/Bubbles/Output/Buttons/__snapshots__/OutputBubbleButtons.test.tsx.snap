// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Output Bubble Buttons snapshot tests should render 1`] = `
<Stack
  horizontal={true}
>
  <StackItem
    style={
      {
        "paddingTop": "5px",
      }
    }
  >
    <InsertButton
      sqlQuery=""
    />
  </StackItem>
  <StackItem>
    <CopyButton
      sqlQuery=""
    />
  </StackItem>
  <StackItem>
    <FeedbackButtons
      sqlQuery=""
    />
  </StackItem>
  <StackItem>
    <MoreButton />
  </StackItem>
</Stack>
`;
